public/app/features/logs/components/LogRow.tsx

```diff
@@ -79,20 +73,13 @@ class UnThemedLogRow extends PureComponent<Props, State> {
     showDetails: false,
   };
 
-  toggleContext = (method: string) => {
-    const { datasourceType, uid: logRowUid } = this.props.row;
-    reportInteraction('grafana_explore_logs_log_context_clicked', {
-      datasourceType,
-      logRowUid,
-      type: method,
-    });
+  debouncedContextClose = debounce(() => {
+    this.setState({ showContext: false });
+  }, 3000);
```

Comment:
Could you please explain why do we have this here? 

Also, more generic question - considering that we have log context modal (which true/false state is in `Logs` component) , why do we need to have this state here as well?

Reply:
https://user-images.githubusercontent.com/8092184/231766750-9ff01296-cc42-4a5a-b569-68f1b04227bd.mov

I like that the LogsContainer still scrolls to the middle. The state setting is to highlight the logline that was opened after LogContext was closed.

Do you still think we should change this behavior?

Reply:
For me personally it is slightly confusing/distracting. I am getting fresh new view (modal) and I can't see/interact with lines behind the modal, so I would expect them to stay in the same position. Also, downside for me it that it complicates the code and as I mentioned, I don't see much benefit in it. But maybe others have different opinion. 🙂

Reply:
Okay, let's remove the scrolling for now - I can see that point. I'll also sync with @niat22 this afternoon.